src/Blocks/BlockComparator.h

```diff
@@ -57,7 +80,16 @@ class cBlockComparatorHandler :
 	{
 		a_BlockType = m_BlockType;
 		a_BlockMeta = cBlockRedstoneRepeaterHandler::RepeaterRotationToMetaData(a_Player.GetYaw());
-		return true;
+
+		Vector3i Pos{ a_BlockX, a_BlockY, a_BlockZ };
+
+		bool Result = a_Player.GetWorld()->DoWithChunkAt(Pos, [&](cChunk & a_Chunk)
```

Comment:
Is there a reason not to just put this check in `cItemHandler::OnPlayerPlace` or perhaps `cItemHandler::GetBlocksToPlace`? It seems like this check is a fundamental part of the placement and not specific to one or two items.

Reply:
I agree... I didn't thought of that, I will move these check to cItemHandler::GetBlocksToPlace